.gitignore

```
@@ -42,6 +42,9 @@ share/python-wheels/
 *.egg
 MANIFEST
 
+# Poetry requirements / packaging
+pyproject.toml
```

```suggestion
/pyproject.toml
```

b/c Anthony is a bad man.